presto-main-base/pom.xml

```diff
@@ -456,6 +455,11 @@
             <groupId>io.netty</groupId>
             <artifactId>netty-transport</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>com.facebook.airlift</groupId>
+            <artifactId>http-client</artifactId>
```

Comment:
we dont want airlift dependency for presto-main-base. this is for java version update.
cc: @ZacBlanco

Reply:
Thanks for the ping; To elaborate We can have some airlift dependencies inside of `presto-main-base`, such as `log` and `configuration`, but we need to keep those which rely on jetty (directly or transitively) outside of `presto-main-base` so that we can maintain support for Presto-On-Spark on Java 8.

A good way to verify right now if the airlift module is compatible is by going to https://github.com/prestodb/airlift and checking if the `pom.xml` for that particular module has a `<project.build.targetJdk>` set to 17. If it does, then it cannot be included in `presto-main-base` for the time being.